traitlets/config/configurable.py

```diff
@@ -365,9 +365,9 @@ def c(s):
                 # Truncate help to first line + "See also Original.trait"
                 if trait.help:
                     lines.append(c(trait.help.split('\n', 1)[0]))
-                lines.append('#  See also %s.%s' % (defining_class.__name__, name))
+                lines.append('#  See also: %s.%s' % (defining_class.__name__, name))
 
-            lines.append('# c.%s.%s = %s' % (cls.__name__, name, default_repr))
+            lines.append('#c.%s.%s = %s' % (cls.__name__, name, default_repr))
```

Comment:
Let's keep the space. It's generally [recommended](https://www.python.org/dev/peps/pep-0008/#block-comments) to have a space between the comment symbol and the comment itself. Plus, editors that have a 'comment/uncomment' action generally deal with this in a single keystroke as well.